### PR TITLE
Add CI for kubernetes-sigs/descheduler

### PR DESCRIPTION
**Pull request by ingvagabund**
Depends on https://github.com/kubernetes-sigs/descheduler/pull/297

FYI @damemi

### CODE REVIEW

File: config/jobs/kubernetes-sigs/descheduler/OWNERS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- ingvagabund
+- damemi
```
Comment (ingvagabund) [inline config/jobs/kubernetes-sigs/descheduler/OWNERS:5]: @seanmalloy do you want to be approver as well?

File: config/tests/testgrids/config_test.go [hunk 2]
```diff
@@ -351,6 +351,7 @@ var noPresubmitsInTestgridPrefixes = []string{
 	"GoogleCloudPlatform/k8s-multicluster-ingress",
 	"kubeflow/pipelines",
 	"kubernetes-sigs/cluster-capacity",
+	"kubernetes-sigs/descheduler",
```
Comment (cblecker) [inline config/tests/testgrids/config_test.go:354]: Why no testgrid? We are trying to whittle this list down

Comment (ingvagabund) [reply config/tests/testgrids/config_test.go:354]: I copy-pasted `kubernetes-sigs/scheduler-plugins`. I have no objections to put it some place else. What's the better place to put it in?

Comment (cblecker) [reply config/tests/testgrids/config_test.go:354]: https://testgrid.k8s.io/sig-scheduling

**Comment by ingvagabund** [comment]
@cblecker updated PTAL

File: config/jobs/kubernetes-sigs/descheduler/OWNERS (hunk 1 above)
Comment (seanmalloy) [reply config/jobs/kubernetes-sigs/descheduler/OWNERS:5]: Might want to add @aveshagarwal to the list of approvers

File: config/jobs/kubernetes-sigs/descheduler/descheduler-presubmits.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    decorate: true
+    path_alias: sigs.k8s.io/descheduler
+    always_run: true
+    spec:
+      containers:
+      - image: golang:1.13
+        command:
+        - make
+        args:
+        - verify-gofmt
```
Comment (damemi) [inline config/jobs/kubernetes-sigs/descheduler/descheduler-presubmits.yaml:17]: Maybe we should add a blanket `make verify` command to the descheduler makefile (I was surprised to see it doesn't already have one), so we don't have to update test-infra for all the different verify tests. 

I would also prefer to add more of our verify tests to this anyway, and it's pretty common for repos to run an umbrella "verify" test

Comment (ingvagabund) [reply config/jobs/kubernetes-sigs/descheduler/descheduler-presubmits.yaml:17]: Agree. Can you update the Makefile? I will update the PR accordingly.

Comment (damemi) [reply config/jobs/kubernetes-sigs/descheduler/descheduler-presubmits.yaml:17]: Something like https://github.com/kubernetes-sigs/descheduler/pull/307

File: config/jobs/kubernetes-sigs/descheduler/OWNERS (hunk 1 above)
Comment (ingvagabund) [reply config/jobs/kubernetes-sigs/descheduler/OWNERS:5]: @aveshagarwal do you want to be approver?

**Pull request merged**